src/v1.js

```diff
@@ -586,6 +587,59 @@ builder.declare({
   res.reply(roles.map(r => roleToJson(r, this)));
 });
 
+/** List role Ids **/
+builder.declare({
+  method:     'get',
+  route:      '/roleids/',
+  query: {
+    continuationToken: /./,
+    limit: /^[0-9]+$/,
+  },
+  name:       'listRoleIds',
+  input:      undefined,
+  output:     'list-role-ids-response.yml',
+  stability:  'stable',
+  title:      'List Role IDs',
+  description: [
+    'If no limit is given, the roleIds of all roles are returned. Since this',
+    'list may become long, callers can use the `limit` and `continuationToken`',
+    'query arguments to page through the responses.',
+  ].join('\n'),
+}, async function(req, res) {
+  let hashids = new Hashids();
+  let continuationToken = parseInt(req.query.continuationToken, 10);
```

Comment:
If the continuation token is not defined, then this will return `NaN`. Let's do:

```js
let continuationToken = parseInt(req.query.continuationToken, 10) || undefined;
```